app.js

```diff
@@ -1,13 +1,30 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
 const { getEndpoints } = require("./controllers/endpoint.controller");
+const { getArticlesById } = require("./controllers/articles.controller");
 
 const app = express();
-
+app.use(express.json());
```

Comment:
Dont need this yet, don't add code unless you need it to pass the test in future or we can end up with bloated codebases